src/app/components/BCP47Tag.svelte

```diff
@@ -1,11 +1,19 @@
 <script lang="ts">
   import bcp47Parser from "bcp-47";
+  import worker from "../spawn-worker";
 
   const TAG_INPUT_ID = "tag-input";
 
   let bcp47Tag = "";
+  let invalidTag = true;
   $: schema = bcp47Parser.parse(bcp47Tag);
   $: invalidTag = bcp47Tag !== "" && schema.language === null;
+  $: {
+    if (!invalidTag) {
```

Comment:
I almost mentioned this during our pair session, but could you **flip* this boolean variable? Instead of

```
invalidTag = true; // the tag is not valid
```

It's simpler to think of:

```
validTag = false
```

(this suggestion fixes a bug where an empty tag was considered a valid tag):

```suggestion
  let validTag = false;
  $: schema = bcp47Parser.parse(bcp47Tag);
  $: validTag = bcp47Tag !== "" && schema.language != null;
  $: {
    if (validTag) {
```

I believe there's code later on that makes use of `invalidTag`, so you'll have to update that too!